afterpay/src/main/kotlin/com/afterpay/android/CancellationStatus.kt

```diff
@@ -6,5 +6,5 @@ enum class CancellationStatus {
     INVALID_CHECKOUT_URL,
     NO_CHECKOUT_HANDLER,
     NO_CONFIGURATION,
-    LANGUAGE_NOT_SUPPORTED,
+    LANGUAGE_NOT_SUPPORTED
```

Comment:
Are all these trailing commas deleted manually or by an automatic fixer?
Kotlin seems to support it (at least on 1.7.x version)
https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Ktlint was one of the updates made which added a new rule for trailing commas.

Reply:
I have updated the rule and added all of the commas. As the previous version of ktlint that was used didn't have any rule for trailing commas, many locations have had a comma added.

Changed in 6554a73

Reply:
Awesome :)